acstools/acszpt.py

```diff
@@ -268,116 +257,84 @@ def _check_date(self, fmt='%Y-%m-%d'):
                 result = ('The observation date cannot occur '
                           'before ACS was installed '
                           f'({self._acs_installation_date.strftime(fmt)})')
+            if dt_obj > self._end_table_date:
```

Comment:
Maybe we should add a few more date checks here. HRC has been offline since Jan 27, 2007, and WFC was not operational between Jan 27, 2007 and May 24, 2009, so we could prevent the user from getting results for those dates. Do you agree, @gsanand and/or @rjavila ?

Reply:
I don't think that's necessary. I can't imagine why anyone would request zeropoints for data that don't exist.

Reply:
Ok, sounds good. The existing date checking is good then.